Hover.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name = 'Hover'
-    spec.version = '1.5.0'
+    spec.version = '1.6.0'
```

Comment:
This can't be a `minor` change as we're breaking the API, this must be a `major`: `2.0.0`